src/btree/bt_sync.c

```diff
@@ -40,6 +40,9 @@ __sync_checkpoint_can_skip(WT_SESSION_IMPL *session, WT_REF *ref)
      *     skip them,
      * 5. there's already an address for every disk block involved.
      */
+    // TODO - we may want to disable during shutdown and recovery as well.
+    if (F_ISSET(S2C(session), WT_CONN_RTS_RUN))
+        return (false);
```

Comment:
We will need to asses the perf impacts of this - we are now dealing with all btrees during checkpoint RTS.

Reply:
The existing session flag can work here. We are performing the checkpoint as part of the RTS session itself.

Reply:
Sure, done in 0f618f663b588be2e87ea74440ae4b073a200a8f. I reverted all changes related to the new flag `WT_CONN_RTS_RUN`.